sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -374,6 +389,21 @@ RestClient createClient() throws IOException {
           throw new IOException("Can't load the client certificate from the keystore", e);
         }
       }
+      if (getMaxRetryTimeoutMillis() != null) {
+        restClientBuilder.setRequestConfigCallback(
+            new RestClientBuilder.RequestConfigCallback() {
```

Comment:
No need to UTest that the timeout is correctly set as it is a restClient built in mechanism.